prediction_market_agent_tooling/markets/data_models.py

```diff
@@ -34,6 +38,10 @@ class Bet(BaseModel):
     outcome: bool
     created_time: datetime
     market_question: str
+    market_id: HexAddress
```

Comment:
This is a general `Bet` class,  so this should be just a string.

```suggestion
    market_id: str
```

And then you don't have to do `market_id=HexAddress(HexStr(market.id)),` in  `manifold_to_generic_resolved_bet`, because Manifold ID is not a HexAddress